src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -42,47 +53,37 @@
         }
 
         dialog table {
-            border: 1px solid #ccc;
+            background: var(--page-background);
             border-collapse: collapse;
+            border-radius: 4px;
+            box-shadow: inset 0 0 0 1px var(--table-border-color);
+            color: var(--page-color);
             margin: 0 0 1em 0;
             margin-bottom: 1em;
             padding: 0;
-            table-layout: fixed;
         }
 
         dialog table tr {
-            background-color: #f8f8f8;
-            border: 1px solid #ddd;
+            border: 1px solid var(--table-border-color);
             padding: .35em;
         }
 
         dialog table th,
         dialog table td {
+            border: 1px solid var(--table-border-color);
             padding: .625em;
-            text-align: center;
+            text-align: LEFT;
```

Comment:
why uppercasing it ?

Reply:
No reason 😊 This was unintended. Fixed. Thanks.